webapi/src/main/scala/org/knora/webapi/util/TransformData.scala

```diff
@@ -254,8 +260,6 @@ object TransformData extends App {
 
     /**
       * Looks for regions that have the label "test" and changes their label to the value of their `knora-base:hasComment`.
-      *
```

Comment:
Why did you remove the documentation of the required param? Because it is already documented in the abstract super class?